index.html

```diff
@@ -32,7 +32,149 @@ <h2 class="first-name software-dev">I'm a software developer</h2>
   <div class="arrow-flex display-flex">
     <span class="arrow">&or;</span>
 </div>
-</section>      
+</section>
+
+<!-- Recent Work Session -->
+  <h2 class="recent-work-align">My Recent</h2>
+  <h2 class="recent-work-align">Works</h2>
+<div class="dash-flex display-flex">
+<span class="dash">&#8212;</span>
+</div>
+<section class="container-recent">  
+  <div class="grid-project">
+    <div class="multi-post">
+     <h3 class="multi-text-color">Multi-Post Stories</h3>
+     <h3 class="multi-text-color">Gain+Glory</h3>
+     <ul class="language display-flex">
+     <li class="language-background">Ruby on rails</li>
+     <li class="language-background">css</li>
+     <li class="language-background" >JavaScript</li>
+     <li class="language-background">html</li>    
+   </ul>  
+   <button class="see-project">See Project</button>
```

Comment:
### OPTIONAL
- [ ]  Kindly add  the button styling for : Active, Hover and Pressed (see screenshot below)

![Screenshot_54](https://user-images.githubusercontent.com/69638013/165087220-ca342432-ff0f-4660-aa11-fd3205002661.png)
- Please do this for all buttons in this section 👍🏿